docs/writing-docs/doc-blocks.md

```diff
@@ -242,7 +242,8 @@ As an example, if you had the following story:
   paths={[
     'react/button-story-default-docs-code.js.mdx',
     'react/button-story-default-docs-code.ts.mdx',
-    'vue/button-story-default-docs-code.js.mdx'
+    'vue/button-story-default-docs-code.js.mdx',
+    'angular/button-story-default-docs-code.ts.mdx'
```

Comment:
```suggestion
    'angular/button-story-default-docs-code.ts.mdx',
```

Reply:
Fixed with the latest commit. Thank you for checking it :)